espresso-server/app/src/androidTest/java/io/appium/espressoserver/lib/helpers/w3c/state/ActiveInputSources.java

```diff
@@ -14,7 +15,7 @@
  */
 public class ActiveInputSources {
     private Map<String, InputSource> inputSources = new HashMap<>();
-    private static ActiveInputSources globalActiveInputSources;
+    private static Map<String, ActiveInputSources> activeInputSources = new HashMap<>();
```

Comment:
should the mapping be weak?

Reply:
Yeah good thinking. Probably when sessions are deleted we should be removing these.